### PR TITLE
Sets anonymizeIp to true in GA snippet

### LINKED ISSUE #2212: AnonymizeIP for Google Analytics
@coliff writes:

>by default Google Analytics records the full IP address of a user visiting the site, but that full IP address is never available to the Google Analytics property admin. By anonymizing the IP address you can make your site more GDPR-compliant as a full IP address can be defined as publicly identifiable information. There is almost no downsides to this, other than the location of users can be slightly less accurate at a city-level. We already provide info about AnonymizeIP in the docs but I think it might be worthwhile including by default. Worth a discussion....

### PR DESCRIPTION
**Pull request by roblarsen**
closes #2212

### CODE REVIEW

File: dist/doc/html.md [hunk 1]
```diff
@@ -176,6 +176,13 @@ included.
 We use `analytics.js` rather than the newer `gtag.js` as
 [it's faster and supports tasks and plugins](https://github.com/philipwalton/analyticsjs-boilerplate/issues/19#issuecomment-333714370)
 
+Starting with version 8 we, by default, <a href="https://support.google.com/analytics/answer/2763052?hl=en">anonymizse IP addresses</a>.
```
Comment (coliff) [inline dist/doc/html.md:179]: anonymize typo here 🤓 
Also - should use markdown style links, not HTML ones I think

File: dist/doc/html.md [hunk 2]
```diff
@@ -176,6 +176,13 @@ included.
...
+By default Google Analytics records the full IP address of a user visiting the site, but
+that full IP address is never available to the Google Analytics property admin.
+By anonymizing the IP address you can make your site more GDPR-compliant as a
+full IP address can be defined as PII (publicly identifiable information.)
```
Comment (coliff) [inline dist/doc/html.md:183]: sorry another thing... my fault as this was my comment... but this should actually be personally identifiable information (PII)
REF: https://en.wikipedia.org/wiki/Personal_data

File: src/doc/html.md [hunk 3]
```diff
@@ -176,11 +176,11 @@ included.
 We use `analytics.js` rather than the newer `gtag.js` as
 [it's faster and supports tasks and plugins](https://github.com/philipwalton/analyticsjs-boilerplate/issues/19#issuecomment-333714370)
 
-Starting with version 8 we, by default, <a href="https://support.google.com/analytics/answer/2763052?hl=en">anonymizse IP addresses</a>.
+Starting with version 8 we, by default, [anonymize IP addresses](href="https://support.google.com/analytics/answer/2763052?hl=en).
 By default Google Analytics records the full IP address of a user visiting the site, but
 that full IP address is never available to the Google Analytics property admin.
 By anonymizing the IP address you can make your site more GDPR-compliant as a
-full IP address can be defined as PII (publicly identifiable information.)
+full IP address can be defined as PII (personally identifiable information.)
```
Comment (coliff) [inline src/doc/html.md:184]: sorry, last nit-pick and then we're good. There's 2 line breaks after this section and should be just 1 to be consistent.

Comment (roblarsen) [reply src/doc/html.md:184]: nit-picking is why we're here 👍

**Comment by TheDancingCode** [comment]
Another level of privacy might be to respect the DoNotTrack signal of browsers.

In that case, the code would be wrapped in a conditional:
```js
var dnt = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
if (dnt !== 'yes' && dnt !== '1') {
    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
    ga('create', 'UA-XXXXX-Y', 'auto'); ga('set', 'anonymizeIp', true); ga('set','transport','beacon'); ga('send', 'pageview');
}
```

**Comment by coliff** [comment]
thanks @TheDancingCode - the DoNotTrack  has actually been deprecated now.

> DNT is not widely adopted by the industry, with companies citing the lack of legal mandates for its use, as well as unclear standards and guidelines for how websites are to interpret the header. Thus, it is not guaranteed that enabling DNT will actually have any effect at all. The W3C disbanded its DNT working group in January 2019, citing insufficient support and adoption. Apple discontinued support for DNT the following month.

https://en.wikipedia.org/wiki/Do_Not_Track

**Pull request merged**